generator/template/ready-to-use-section-types/sections/forms/ResturantMenu.vue

```diff
@@ -0,0 +1,634 @@
+      </div>
+
+      <div class="flex flex-col items-start justify-start mt-8 pt-8 border-t">
+        <h3 class="text-lg font-semibold mb-4">{{ $t("RestaurantMenu.taxSettings") || "Tax Settings" }}</h3>
+
+        <div class="flex flex-col items-start justify-start mt-4">
+          <div class="flex items-center mb-2">
+            <input v-model="settings[0].enableTax" type="checkbox" id="enableTax" class="h-5 w-5 mr-2" />
+            <label for="enableTax" class="mr-4 font-medium">
+              {{ $t("RestaurantMenu.enableTax") || "Enable Tax" }}
+            </label>
+          </div>
+          <span class="text-xs text-Gray_800 mb-2">
+            {{ $t("RestaurantMenu.enableTaxDesc") || "Enable tax calculation for customer orders" }}
+          </span>
+        </div>
+
+        <div v-if="settings[0].enableTax" class="flex flex-col items-start justify-start mt-4">
+          <label class="mr-4 font-medium">
+            {{ $t("RestaurantMenu.taxRate") || "Tax Rate (%)" }}
+          </label>
+          <input v-model.number="settings[0].taxRate" type="number" min="0" max="100" step="0.01" placeholder="10.00"
+            :class="sectionsStyle.input" />
+          <span class="text-xs text-Gray_800">
+            {{ $t("RestaurantMenu.taxRateDesc") || "Percentage of tax applied to orders" }}
+          </span>
+        </div>
+      </div>
+
+      <div class="flex flex-col items-start justify-start mt-4">
+        <label class="mr-4 font-medium">{{ $t("RestaurantMenu.cssClasses") }}</label>
+        <span class="text-xs text-Gray_800">{{ $t("RestaurantMenu.cssClassesDesc") }}</span>
+        <input v-model="settings[0].classes" type="text" placeholder="CSS Classes" :class="sectionsStyle.input" />
+      </div>
+
+      <div class="flex flex-col items-start justify-start mt-4">
+        <label class="mr-4 font-medium">{{ $t("RestaurantMenu.backgroundColor") || "Background Color" }}</label>
+        <input v-model="settings[0].backgroundColor" type="color" :class="sectionsStyle.input" />
+        <span class="text-xs text-Gray_800">{{ $t("RestaurantMenu.backgroundColorDesc") }}</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import UploadMedia from "@geeks.solutions/nuxt-sections/lib/src/components/Medias/UploadMedia.vue";
+import FieldSets from "@geeks.solutions/nuxt-sections/lib/src/components/SectionsForms/FieldSets.vue";
+import { sectionsStyle, scrollToFirstError } from "@/utils/constants";
+import 'vue-select/dist/vue-select.css';
+import { v4 as uuidv4 } from 'uuid';
+
+export default {
+  name: 'RestaurantMenu',
+  components: { FieldSets, UploadMedia },
+  props: {
+    selectedLang: {
+      type: String,
+      default: 'en'
+    },
+    selectedMedia: {},
+    selectedMediaIndex: {
+      type: Number,
+      default: -1
+    },
+    locales: {
+      type: Array,
+      default() {
+        return []
+      }
+    },
+    mediaFields: [
```

Comment:
There media fields used are icon, image and logo. no media field with name `media` exist in the settings.

Note: To have media content linking working as expected, it is recommended to have one additional field in settings ie: `medias` that is a list of the medias used in the section, which will be filtered out once an media gets added or removed.

Then in this `mediaFields` that is responsible for the content linking feature, you can have one field only that has name `medias`. Instead of putting adding 3 for logo, icon and image